src/category_theory/concrete_category/bundled_hom.lean

```diff
@@ -85,6 +85,44 @@ has_forget₂.mk'
   @map
   (by intros; apply heq_of_eq; apply h_map)
 
+variables {d : Type u → Type u}
+variables (hom)
+
+/--
+The `hom` corresponding to first forgetting along `F`, then taking the `hom` associated to `c`.
+-/
+@[reducible] def map_hom (F : Π {α}, d α → c α) : Π ⦃α β : Type u⦄ (Iα : d α) (Iβ : d β), Type u :=
+λ α β iα iβ, hom (F iα) (F iβ)
+
+/--
+Construct the `bundled_hom` induced by a map between type classes.
+This is useful for building categories such as `CommMon` from `Mon`.
+-/
+def map (F : Π {α}, d α → c α) : bundled_hom (map_hom hom @F) :=
+{ to_fun := λ α β iα iβ f, 𝒞.to_fun (F iα) (F iβ) f,
+  id := λ α iα, 𝒞.id (F iα),
+  comp := λ α β γ iα iβ iγ f g, 𝒞.comp (F iα) (F iβ) (F iγ) f g,
+  hom_ext := λ α β iα iβ f g h, 𝒞.hom_ext (F iα) (F iβ) h }
+
+section
+omit 𝒞
+/--
+We used the empty `parent_projection` class to label functions like `comm_monoid.to_monoid`,
+which we would like to use to automatically construct `bundled_hom` instances from.
+
+Once we've set up `Mon` as the category of bundled monoids,
+this allows us to set up `CommMon` by defining an instance `parent_project (comm_monoid.to_monoid)`.
+-/
+class parent_projection (F : Π {α}, d α → c α)
+end
+
+instance (F : Π {α}, d α → c α) [parent_projection @F] : bundled_hom (map_hom hom @F) := map hom @F
```

Comment:
````
/- UNUSED ARGUMENTS: -/
#print category_theory.bundled_hom.category_theory.bundled_hom /- argument 6: [_inst_1 : category_theory.bundled_hom.parent_projection F] -/
````
Also, dodge name?

Reply:
Fixed (using `@[nolint unused_arguments]`), and given a name.